extensions/Dockerfile

```diff
@@ -32,3 +32,14 @@ RUN createrepo_c /usr/share/rpm-ostree/extensions/
 ## Final container that has the extensions repo dir
 FROM registry.access.redhat.com/ubi9/ubi:latest
 COPY --from=builder /usr/share/rpm-ostree/extensions/ /usr/share/rpm-ostree/extensions/
+
```

Comment:
probably need to add a comment above this block that mentions where we are writing `extensions.json` is bind mounted in during the build and what we are doing here will be used to generated the meta.json entry for extensions. 

you can link to the COSA code that does the `--volume` part for added clarity